src/components/Modals/FiatRamps/views/RampsList.tsx

```diff
@@ -52,6 +53,16 @@ export const RampsList: React.FC<RampsListProps> = ({ setFiatRampProvider }) =>
                   <Box textAlign='left' ml={2}>
                     <Text fontWeight='bold' translation={fiatRampConfig.label} />
                     <Text translation={fiatRampConfig.info ?? ''} />
+                    {fiatRampConfig.tags?.map(tag => (
```

Comment:
two tiny bits of technical feedback. in this particular case the second is not as relevant, but a good pattern to practise

1. plz add a key on this, future ramps (mt pelerin) will have multiple tags and cause a react warning
2. in general, don't map in components - memoize and insert the memoized elements. this is not an issue here, but best-practice for larger components for perf reasons.